custom-namespaces.yaml

```
@@ -421,3 +421,92 @@ sync:
       type: ping_explore
       views:
         base_view: sync
+monitoring:
+  glean_app: false
+  owners:
+    - jklukas@mozilla.com
+    - ascholtz@mozilla.com
+    - wlach@mozilla.com
+  pretty_name: Monitoring
```

```suggestion
  pretty_name: Monitoring (Data Pipeline)
```

Given the visibility of "Operational Monitoring" as a concept, it seems like we should provide a bit more context about what this is.